File: backend/dotnet/ClinicAppointments.Db/Entities/Secretary.cs
namespace ClinicAppointments.Db.Entities;

public class Secretary
{
    public Guid Id { get; set; }
    public required string Name { get; set; }
    public required string Email { get; set; }
    public required string Password { get; set; }
    public required string Phone { get; set; }
    public DateOnly Birthdate { get; set; }
    public required string Cpf { get; set; }
    public required string Cnpj { get; set; }
}

File: tests2/src/lib/api/validation/Secretary.ts
import { z } from 'zod'

export const SecretarySchema = z.strictObject({
  id: z.uuidv4(),
  name: z.string(),
  email: z.email(),
  phone: z.string(),
  birthdate: z.coerce.date(),
  cpf: z.string(),
  cnpj: z.string(),
})

export type Secretary = z.infer<typeof SecretarySchema>
